app/views/staff/events/configuration.html.haml

```diff
@@ -1,3 +1,6 @@
+- if params[:controller].include?("staff") && user_signed_in?
+  = render partial: "layouts/nav/staff_subnav"
```

Comment:
You could cut down on template code and wrap this in a helper method like "render_staff_subnav". Not really necessary though.